ReserveSpot.Services.Api/Startup.cs

```diff
@@ -0,0 +1,58 @@
+// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+
+namespace ReserveSpot.Services.Api
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration) =>
+                    Configuration = configuration;
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+            services.AddSwaggerGen(options =>
+            {
+                var OpenApiInfo = new OpenApiInfo
+                {
+                    Title = "ReserveSpot.Services.Api",
+                    Version = "v1"
+                };
+
+                options.SwaggerDoc(
+                    name: "v1",
+                    info: OpenApiInfo);
+            });
+        }
+
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+                app.UseSwagger();
+                app.UseSwaggerUI(options =>
```

Comment:
Single & Multi-line code to be separated with new line.

Reply:
Seperated single and Multi-line code with new line